src/SFA.DAS.EmployerIncentives/Interfaces/IApplicationService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.InnerApi.Requests.IncentiveApplication;
+using SFA.DAS.EmployerIncentives.InnerApi.Responses;
+
+namespace SFA.DAS.EmployerIncentives.Interfaces
+{
+    public interface IApplicationService
```

Comment:
The is the application service so deals with applications so there's no need to include the IncentiveApplication in the method name.  Can it be just IApplicationService.Create and IApplicationService.Update etc... ?

IApplicationService.GetApprenticeApplications that returns a GetApplicationsResponse is confusing.  Not sure if this ticket is to tidy this up?

Also why CreateIncentiveApplicationRequestData and UpdateIncentiveApplicationRequestData?  They are almost the same and surely it's just an IncentiveApplication?  Probably outside the scope of this ticket though since the inner api would need tidying up first.